app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :email, String
+  property :password, Text
+
+  validates_presence_of :email
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.